### ISSUE TITLE
StructuredList is not accessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6894]
## Environment

> Operating system

Windows 10 latest

> Browser

Edge, Firefox

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description
The checkmarks indicating a selection made are not visually accessible

Firefox
![image](https://user-images.githubusercontent.com/40970507/93945266-363ac200-fcfc-11ea-82ca-fe7e84dd08d5.png)

Edge
![image](https://user-images.githubusercontent.com/40970507/93945273-3aff7600-fcfc-11ea-9153-a3a9344ba3fc.png)
